src/graph/transform/to_block.cc

```diff
@@ -189,6 +358,8 @@ std::tuple<HeteroGraphPtr, std::vector<IdArray>> ToBlock<kDGLCUDA, int64_t>(
 
 #endif  // DGL_USE_CUDA
 
+// This code is called in `dgl.transform.to_block`, which is tested in
```

Comment:
remove this and apply the snippet on the top.

Reply:
removed